.travis.yml

```diff
@@ -13,7 +13,9 @@ cache:
 
 env:
   matrix: SYMFONY_VERSION=3.2.*
-  global: SYMFONY_PHPUNIT_DIR=.phpunit SYMFONY_PHPUNIT_REMOVE="symfony/yaml"
+  global:
+    - SYMFONY_DEPRECATIONS_HELPER=2
```

Comment:
@wouterj is the value of this env variable the number of deprecations we accept? if so this will fix the build until monolog does not trigger deprecations anymore.